libs/s25main/gameData/GameConsts.h

```diff
@@ -20,10 +20,16 @@
 #include "helpers/EnumArray.h"
 #include "helpers/make_array.h"
 #include "gameTypes/GameSettingTypes.h"
+#include <chrono>
 #include <limits>
 
+using namespace std::chrono_literals;
```

Comment:
Just a word of caution here: `using namespace` in global namespace in a header is discouraged. However this one is fine by me.